scripts/prepare_sector_network.py

```diff
@@ -184,6 +184,55 @@ def H2_liquid_fossil_conversions(n, costs):
     )
 
 
+def add_with_water_network(n, costs):

```

Comment:
Why "with" water network? How about
```suggestion
def add_water_network(n, costs):

```

Reply:
agree!